<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>1081</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(170, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <widget class="QLineEdit" name="quantity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 143, 62, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="unitprice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Unit Price</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="datein">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="threshold">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>threshold</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>881</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Threshold</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>datein</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>dateout</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="description">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(46,82,101,200);
border-bottom:2px solid rgba(46,82,101,200);
border-top:2px solid rgba(46,82,101,200);

color:rgba(0,0,0,240);
paddin-bottom:7x;
font-size:10pt; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 143, 62, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Retourner vers le main</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="des.qrc"/>
 </resources>
 <connections/>
</ui>
